### ISSUE TITLE
problem with import of `@internationalized/date` after upgrade to `@ark-ui/react@3.6.1`

### ISSUE THREAD

**Issue by arekmaz** [chakra-ui/ark#2704]
### Description

I'm trying to upgrade my remix vite project from `@ark-ui/react@3.5.0`  to the latest 3.6.1,

I'm getting the following error:
```
import { parseTime as $fae977aafc393c5c$export$c9698ec7f05a07e1 } from '../../node_modules/@internationalized/date/dist/string.js';
         ^^^^^^^^^
SyntaxError: Named export 'parseTime' not found. The requested module '../../node_modules/@internationalized/date/dist/string.js' is a CommonJS module, which may not support all module.exports as named exports.
CommonJS modules can always be imported via the default export, for example using:

import pkg from '../../node_modules/@internationalized/date/dist/string.js';
const { parseTime: $fae977aafc393c5c$export$c9698ec7f05a07e1 } = pkg;
```

it seems to come from this recently added [use-date-picker file](https://github.com/chakra-ui/ark/blob/7f0d6db3a07bcf07f87d03b361f3a01778c5210d/packages/react/src/components/time-picker/use-time-picker.ts#L1)

I suspect the reason might be because `@internationalized/date`  module direct usage was recently added, but it's not listed as a dependency or peer dependency of the project.

### Link to Reproduction (or Detailed Explanation)

https://github.com/arekmaz/arek-ui/tree/ark-3.6.1-broken-import-repro

### Steps to Reproduce

1. clone https://github.com/arekmaz/arek-ui
2. enter the directory, run `pnpm i`
3. `npm run dev` is running correctly, after visiting http://localhost:3000, the app is working
4. run `pnpm i @ark-ui/react@3.6.1`, or checkout the branch `ark-3.6.1-broken-import-repro` and `pnpm i`
5. `npm run dev` might be running, but after visiting http://localhost:3000, the before mentioned error is shown

### Ark UI Version

3.6.1

### Framework

- [X] React
- [ ] Solid
- [ ] Vue

### Browser

Brave, Google Chrome

### Additional Information

I suspect it might not be react only, if the problem really is listing the `@internationalized/date` package, it was not listed in any framework package

**Comment by lilouartz** [comment]
I can replicate the same issue.

**Comment by cschroeter** [comment]
Can you declare @internationalized/date as a dependency yourself to resolve the issue

**Comment by cschroeter** [comment]
@arekmaz Issue should be resolved in `3.6.2` Please re-open if the issue persists.

**Comment by arekmaz** [comment]
it's fixed, thanks
